opendmarc/opendmarc.c

```diff
@@ -775,18 +778,18 @@ dmarcf_freearray(char **a)
 }
 
 /*
-**  DMARCF_MKARRAY -- convert a comma-separated string into an array
+**  DMARCF_MKARRAY -- convert a delimiter-separated string into an array
 **
 **  Parameters:
 **  	str -- input string
-**  	array -- output array
+**  	delim -- delimeter string
```

Comment:
Given the definition of strtok_r(), this should actually be "set of delimiter characters".